src/host/CommandListPopup.cpp

```diff
@@ -151,7 +151,7 @@ NTSTATUS CommandListPopup::_deleteSelection(CookedRead& cookedReadData) noexcept
 {
     try
     {
-        auto& history = cookedReadData.History();
+        auto history = cookedReadData.History();

```

Comment:
Because `cookedReadData.History()` returns a `CommandHistory&` each of the autos in this file need to be `auto&` to avoid copying the command history.

Reply:
Oh yes...